ginga/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = 'ginga' %}
 {% set version = '2.5.20160627100500' %}
```

Comment:
We need to change this timestamp too, otherwise we'll overwrite the original package (and no one will get the update).